### DISCUSSION TITLE
Como funciona o Javascript na aplicacao de fato ?

### DISCUSSION THREAD

**Discussion by washonrails** [washonrails/fornite_landpage#4] [Q&A] [answered]
Eu nao entendo muito bem as coisas no back-end podem explicar ?

**Comment by washonrails** [comment]
Basicamente e o processo dos dados , que vem do front-end . No back-end tudo tem de ser mais seguro , e ergonomico

**Comment by iamwash** [comment, marked as answer]
SImplesmente para fazer animacoes no site
